openedx/core/djangoapps/bookmarks/tests/test_tasks.py

```diff
@@ -164,3 +164,19 @@ def test_update_xblocks_cache(self, course_attr, expected_sql_queries):
 
         with self.assertNumQueries(3):
             _update_xblocks_cache(course.id)
+
+    def test_update_xblocks_cache_with_display_name_none(self):
+        """
+        Test that the xblocks data is persisted correctly with display_name=None.
+        """
+        ItemFactory.create(parent_location=self.sequential_2.location, category='vertical', display_name=None)
```

Comment:
@muzaffaryousaf We should check there is a valid `XBlockCache` item for this XBlock.